package.json

```diff
@@ -290,7 +290,7 @@
     "fancy-log": "^1.3.3",
     "fast-glob": "^3.2.2",
     "fs-extra": "^8.1.0",
-    "ganache-cli": "^6.12.1",
+    "ganache": "^v7.0.0-rc.0",
```

Comment:
We're cool with using a pre-release I'm guessing?

Reply:
yes